setup.py

```diff
@@ -1,4 +1,5 @@
 from typing import List
+
```

Comment:
Confirm if this was done by the formatter, if so keep this change.

Reply:
yap, it was